CRM/Dashlet/Page/Blog.php

```diff
@@ -87,7 +87,7 @@ protected function getData() {
           return $new_data;
         }
       }
-      return unserialize($cache->data);
+      return CRM_Core_BAO_Cache::getItem('dashboard', 'newsfeed');
```

Comment:
Shall we switch over to the preferred signature

```
 Civi::cache()->get(''dashboard_newsfeed'); 
```
We would also need to update the set function

```
 Civi::cache()->set(''dashboard_newsfeed', $value, (60 * 60 * 24 * self::CACHE_DAYS)); 
```

Reply:
(this is the set function we are calling)

```
  /**
   * @param string $key
   * @param mixed $value
   * @param null|int|\DateInterval $ttl
   * @return bool
   * @throws \Psr\SimpleCache\InvalidArgumentException
   */
  public function set($key, $value, $ttl = NULL) {
    CRM_Utils_Cache::assertValidKey($key);
    $this->_cache[$key] = $this->reobjectify($value);
    $this->_expires[$key] = CRM_Utils_Date::convertCacheTtlToExpires($ttl, self::DEFAULT_TIMEOUT);
    return TRUE;
  }
```

Reply:
ah hmm, digging further, this seems to work:
```
Civi::cache('community_messages')->set('dashboard_newsfeed', $value, (60 * 60 * 24 * self::CACHE_DAYS));
```
I don't see where the default cache is defined, but `Civi/Core/Container.php` seems to define at least those types of caches, as being long-term caches (giving priority to memcache/apc, sqlgroup, and otherwise arraycache).

I also validated that when setting a ttl, and then changing the `expired_date` to a past date, and running `cvapi Job.cleanup`, the cache is indeed flushed.